app/services/idv/profile_maker.rb

```diff
@@ -47,7 +47,9 @@ def save_profile(
     private
 
     def set_idv_level(in_person_verification_needed:, selfie_check_performed:)
-      if in_person_verification_needed
+      if in_person_verification_needed && IdentityConfig.store.in_person_proofing_enforce_tmx
```

Comment:
Slight refactor to help clarify what's going on with this check (feel free to ignore)
```ruby
  if in_person_verification_needed
    if IdentityConfig.store.in_person_proofing_enforce_tmx
      :in_person
    else
      :legacy_in_person
  elsif ...
```

Reply:
I originally had it coded that way but refactored it. I believe elsif is more performant over a nested if. I am happy to change it if nested is preferred.

Reply:
I don't feel strongly about any of this, but...

I had actually initially gotten hung up on line 52, because I think the `&& !IdentityConfig.store.in_person_proofing_enforce_tmx` bit is logically unnecessary—if it weren't false, it would have matched on line 50. _But_ I think having it anyway makes it a tiny bit clearer what is going on (Jack's suggestion does the same), so I didn't comment.

I'm academically curious to experiment with how nested conditionals compare to inlined ones, but I wouldn't worry too much about that level of optimization here.

tl;dr, I don't have a strong opinion and will support whatever you think is most readable here.

Reply:
Okay, for readability I am going with Jack's suggestion as that is how I originally wrote it as well.  See commit c08407d5368d8e646d9fc411c349c38d5e6c4645

Reply:
It's an edge case, but I _think_ we should probably also check that `FeatureManagement.proofing_device_profiling_decisioning_enabled?` is returning true before marking as non-legacy.

Reply:
@matthinz Just curious but what does that value represent?

Reply:
Matt,

Jack and I talked through your suggestion. We agree with it! We are going to implement it in a different PR as we thought we had better inspect all instances of `IdentityConfig.store.in_person_proofing_enforce_tmx` to evaluate if `FeatureManagement.proofing_device_profiling_decisioning_enabled?` should also be considered. That task seemed out of scope for this ticket and we didn't want it to get overlooked here. I will attach the ticket and PR soon.